### PR TITLE
Added svg background to main element

### PR DESCRIPTION
**Pull request by kkatoch89**
## Description

Added svg background. Background has svg patterns of typical shopping items.

## Related Issue

#14 

## Acceptance Criteria

- Create a crisp clean background
- Make sure the background doesn't take user's attention away from the main aspect of the app

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |

### Before

beforeBackground

### After

afterBackground

## Testing Steps / QA Criteria

To test the feature, use one of the following ways:

1. Follow the [preview link](https://tcl-22-shopping-list--pr53-share-token-d4ye5443.web.app)
2. Launch the app locally from your terminal:
- Pull down this branch with `git pull origin kk-svgBackground` and check that branch out with `git checkout origin kk-svgBackground`
- Run npm start to launch the app in your browser at http://localhost:3000

### CODE REVIEW

**Review by dariatsvetkova — APPROVED** [review]
Looks amazing! Thanks for all the hard work that you put into it, @kkatoch89!

File: src/components/General/Layout.js [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
 
-const StyledWrapper = styled.div`
-  max-width: 1170px;
-  margin: 0 auto;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
+const StyledMain = styled.main`
+  background-image: url(${BackgroundIcons});
+  min-height: 100vh;
```
Comment (laurenmbeatty) [inline src/components/General/Layout.js:7]: Sometimes `100vh` can come back to bite you (especially on mobile). Another way to accomplish this is to give 

height: 100% to `html` `body` and `#root` (the main div of the react app), and then have a `min-height: 100%` here.

**Review by ablizben — APPROVED** [review]
Background looks great! I know you worked really hard on this and we definitely really appreciate it.

**Comment by kkatoch89** [comment]
> 
> 
> Looks amazing! Thanks for all the hard work that you put into it, @kkatoch89!

You helped a lot, so credit to you as well @dariatsvetkova!

File: src/components/General/Layout.js (hunk 1 above)
Comment (kkatoch89) [reply src/components/General/Layout.js]: Sounds good. Thanks for the hot tip! Changes made 👍

Comment (laurenmbeatty) [reply src/components/General/Layout.js]: Sweet...I just learned this tip, myself.

**Review by laurenmbeatty — APPROVED** [review]
LGTM 👍. Pretty background!!!!

**Review by luisaugusto — APPROVED** [review]

**Review by cafloyd — APPROVED** [review]
Looks amazing, love how this turned out -- great work!!

**Pull request merged**